variables.tf

```diff
@@ -115,6 +115,65 @@ variable "packet_user_data" {
   default = ""
 }
 
+/* vultr */
+
+variable "vultr_api_key" {
+  default = ""
+}
+
+variable "vultr_region_id" {
+  /*
+    curl -H "API-Key: $KEY" "https://api.vultr.com/v1/regions/list" \
+      | jq '.[] | select(.name == "New Jersey")'
+  */
+  default = "1"
+}
+
+variable "vultr_plan_id" {
+  /*
+    curl -H "API-Key: $KEY" "https://api.vultr.com/v1/plans/list" \
+      | jq '.[] | select(.price_per_month == "5.00")'
+  */
+  default = "201"
+}
+
+variable "vultr_os_id" {
+  /*
+    curl -H "API-Key: $KEY" "https://api.vultr.com/v1/os/list" \
+      | jq '.[] | select(.name == "Ubuntu 18.04 x64")'
+  */
+  default = "270"
+}
+
+variable "vultr_ssh_key_ids" {
+  /*
+    curl -H "API-Key: $KEY" "https://api.vultr.com/v1/sshkey/list" \
+      | jq '.[].SSHKEYID'
+  */
+  default = [""]
+}
+
+variable "vultr_rate_limit" {
```

Comment:
oh boy, this is very annoying.

Reply:
Let's hide this under the hood, so there is no vultr_rate_limit variable, as default is enough for 3 nodes. But let the module have the parameter for those who need to adjust it.